app/views/blog_posts/index.rb

```diff
@@ -29,8 +29,25 @@ def blog_items
       mustache[:blog_items] ||= @blog_posts.map { |post| blog_item(post) }
     end
 
+    def blogs_filter_options
+      {
+        filter_name: 'theme',
+        options: blogs_filter_options_options
+      }
+    end
+
     protected
 
+    def blogs_filter_options_options
```

Comment:
can this be renamed to just "filter_options"?

blogs_filter_options_options seems very verbose. If renaming, make sure to change in the method call above.